src/modules/Progress/Progress.js

```diff
@@ -184,17 +189,18 @@ class Progress extends Component {
     const ElementType = getElementType(Progress, this.props)
 
     const percent = this.getPercent()
+    const labelElement = this.getLabel()
 
     return (
       <ElementType {...rest} className={classes}>
         <div className='bar' style={{ width: `${percent}%` }}>
           {this.showProgress() && (
             <div className='progress'>
-              { label !== 'ratio' ? `${percent}%` : `${value}/${total}` }
+              { progress !== 'ratio' ? `${percent}%` : `${value}/${total}` }
             </div>
           )}
         </div>
-        {children && <div className='label'>{children}</div>}
+        {!_.isUndefined(children) ? <div className='label'>{children}</div> : labelElement}
```

Comment:
This is the line I was talking about. Can I safely change it to `{children ? <div className='label'>{children}</div> : labelElement}` without the false positives of `0` failing the truthy check?